doc/manual/config-basic.rst

```diff
@@ -0,0 +1,134 @@
+Configuring the system
+======================
+
+Configuration of the judge system is done by logging in as administrator
+to the web interface.
+The default username is ``admin`` with initial password stored in
+``etc/initial_admin_password.secret``.
+
+The general system settings can be accessed under
+*Configuration settings*. Changes take effect immediately.
+
+Setting up users and teams
+--------------------------
+Under *Users* from the homepage you can add user accounts for the
+people accessing your system. There are several roles possible:
+
+- Administrative user: can configure and change everything in DOMjudge.
+- Jury user: can view submissions and judgings. Can view clarification
+  requests and send clarifications. Can rejudge non-correct judgings
+  (submissions judged *correct* can only be rejudged by an administrator.
+- Balloon runner: can only view the balloon queue and mark balloons as
+  delivered.
+- Team member: can view its own team interface and submit solutions
+  (see below).
+- Several system roles: they are for API access. The most imporant
+  one is *judgehost* which means the account credentials can be used by a
+  judgedaemon.
+
+To set up teams, you can start in the *Teams* page and add teams there.
+You then have the option to automatically create a corresponding user
+account that is associated with the team.
+
+It is also possible to use the *Import / Export* page to import ICPC-compatble
+tsv files with teams.
```

Comment:
add link to spec?

Reply:
Where is the spec?

Reply:
https://clics.ecs.baylor.edu/index.php?title=Contest_Control_System_Requirements#teams.tsv